cmd/zetaclientd/keygen_tss.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/zeta-chain/zetacore/zetaclient/zetacore"
 )
 
+// GenerateTss waits for the keygen block height to arrive and generates a new TSS
```

Comment:
We should also consider renaming the function to `GenerateTSS`

Reply:
https://github.com/zeta-chain/node/pull/2411/commits/4f61c380bd8629ea044d7284eabf99c09b447131